### ISSUE TITLE
Bot played JD on first trick

### ISSUE THREAD

**Issue by closedcontour** [tjwilson90/turbo-hearts#2]

**Comment by closedcontour** [comment]
[bad-jd.zip](https://github.com/tjwilson90/turbo-hearts/files/4211606/bad-jd.zip)

**Comment by closedcontour** [comment]
```
e4cdc8ec-9343-4549-ab4f-91e951c34dfb|179|1581903335161|{"type":"deal","north":["QS","9S","8S","7S","6S","4S","KH","4H","JC","TC","7C","3C","2C"],"east":["AS","KS","JS","6H","3H","8D","7D","6D","5D","3D","KC","QC","4C"],"south":["TS","5S","2S","AH","9H","8H","AD","KD","QD","JD","TD","9D","AC"],"west":["3S","QH","JH","TH","7H","5H","2H","4D","2D","9C","8C","6C","5C"],"pass":"across"}
e4cdc8ec-9343-4549-ab4f-91e951c34dfb|181|1581903335161|{"type":"send_pass","from":"north","cards":["QS","9S","8S"]}
e4cdc8ec-9343-4549-ab4f-91e951c34dfb|182|1581903335162|{"type":"send_pass","from":"east","cards":["AS","KC","QC"]}
e4cdc8ec-9343-4549-ab4f-91e951c34dfb|183|1581903335163|{"type":"send_pass","from":"south","cards":["TS","5S","9D"]}
e4cdc8ec-9343-4549-ab4f-91e951c34dfb|184|1581903335163|{"type":"recv_pass","to":"south","cards":["QS","9S","8S"]}
e4cdc8ec-9343-4549-ab4f-91e951c34dfb|185|1581903335163|{"type":"recv_pass","to":"north","cards":["TS","5S","9D"]}
e4cdc8ec-9343-4549-ab4f-91e951c34dfb|186|1581903353368|{"type":"send_pass","from":"west","cards":["QH","4D","2D"]}
e4cdc8ec-9343-4549-ab4f-91e951c34dfb|187|1581903353368|{"type":"recv_pass","to":"west","cards":["AS","KC","QC"]}
e4cdc8ec-9343-4549-ab4f-91e951c34dfb|188|1581903353368|{"type":"recv_pass","to":"east","cards":["QH","4D","2D"]}
e4cdc8ec-9343-4549-ab4f-91e951c34dfb|189|1581903353377|{"type":"charge","seat":"east","cards":[]}
e4cdc8ec-9343-4549-ab4f-91e951c34dfb|190|1581903353378|{"type":"charge","seat":"south","cards":["AH","JD"]}
e4cdc8ec-9343-4549-ab4f-91e951c34dfb|191|1581903353378|{"type":"charge","seat":"north","cards":["TC"]}
e4cdc8ec-9343-4549-ab4f-91e951c34dfb|192|1581903353379|{"type":"charge","seat":"east","cards":[]}
e4cdc8ec-9343-4549-ab4f-91e951c34dfb|193|1581903353379|{"type":"charge","seat":"south","cards":[]}
e4cdc8ec-9343-4549-ab4f-91e951c34dfb|194|1581903361855|{"type":"charge","seat":"west","cards":[]}
e4cdc8ec-9343-4549-ab4f-91e951c34dfb|196|1581903361864|{"type":"play","seat":"north","card":"2C"}
e4cdc8ec-9343-4549-ab4f-91e951c34dfb|197|1581903361865|{"type":"play","seat":"east","card":"4C"}
e4cdc8ec-9343-4549-ab4f-91e951c34dfb|198|1581903361865|{"type":"play","seat":"south","card":"AC"}
e4cdc8ec-9343-4549-ab4f-91e951c34dfb|199|1581903368839|{"type":"play","seat":"west","card":"9C"}
e4cdc8ec-9343-4549-ab4f-91e951c34dfb|200|1581903368848|{"type":"play","seat":"north","card":"TC"}
e4cdc8ec-9343-4549-ab4f-91e951c34dfb|201|1581903368849|{"type":"play","seat":"east","card":"2D"}
e4cdc8ec-9343-4549-ab4f-91e951c34dfb|202|1581903368849|{"type":"play","seat":"south","card":"JD"}
```

**Comment by tjwilson90** [comment]
fixed by 858447cdea18f259c68797ec7a3828d596e56d31
